### PR TITLE
simplify dev setup

### PR DESCRIPTION
**Pull request by r-a303931**
added dev container support so that new developers can get a dev environment up by just installing Docker to their computer or using GitHub codespaces

### CODE REVIEW

File: mysql-init/10-setup-user-permissions.sql
Comment (grioux-neo) [inline mysql-init/10-setup-user-permissions.sql:1]: Why did you remove sql privileges?  Were those permissions for production environment?

**Review by grioux-neo — APPROVED** [review]

File: mysql-init/10-setup-user-permissions.sql
Comment (r-a303931) [reply mysql-init/10-setup-user-permissions.sql:1]: Because in the production database:
https://github.com/cap-md089/evmplus-v6/blob/bf57f09fc3db44d0a49fafeb7b0028b3a6a368dd/docker-compose.yml#L87-L88
and in the development database:
https://github.com/cap-md089/evmplus-v6/blob/bf57f09fc3db44d0a49fafeb7b0028b3a6a368dd/docker-compose.dev.yml#L75-L76
Those values are already set. However, because of how the docker-entrypoint-initdb.d folder works, it caused the mysql server to fail to start up the first time, something that the two of us have not had to experience ever in production or rarely in our own development environments. Removing them from this commit makes the startup consistent, which is necessary for clean environments such as with dev containers.
